net/frr/src/opnsense/mvc/app/controllers/OPNsense/Quagga/Api/Ospf6settingsController.php

```diff
@@ -38,159 +38,32 @@
 
 class Ospf6settingsController extends ApiMutableModelControllerBase
 {
-    static protected $internalModelName = 'OSPF6';
+    static protected $internalModelName = 'ospf6';
```

Comment:
is that a bug? :D

Reply:
no was unused before - get and set were overwritten so a copy and paste issue